aces/Loris
# [timepoint_list] Hiding Candidate Info and View Imaging Datasets buttons

## Brief summary of changes
This PR hides the "Candidate Info" and "View Imaging Datasets" buttons along with the "Create time point" button when the necessary permissions are not present. This was done with the assumption that "View Imaging Datasets" should only be visible if the user has one of the four imaging browser permissions (please correct me if this assumption is wrong). Test plan documentation was edited accordingly. If the user has _none_ of the relevant permissions, the "Actions:" label is also hidden.


#### Testing instructions (if applicable)

1. Access a same-site candidate as the user without any permissions using a link. There should be no "Actions:" label and none of the three buttons "Candidate Info", "View Imaging Datasets", or "Create time point".
2. Access a same-site candidate as the user with only the `data_entry` permission. There should be an "Actions:" label as well as the two buttons "Candidate Info" and "Create time point".
3. Access a same-site candidate as the user with any/all of the imaging browser permissions. There should be an "Actions:" label as well as the "View Imaging Datasets" button (and the other two buttons if the user still has `data_entry` permission).
4. Make sure that clicking on any of the buttons brings you to the right place.

#### Link(s) to related issue(s)

* Resolves #6673
